contentcuration/contentcuration/static/js/edit_channel/utils/mathjaxtosvg.js

```diff
@@ -0,0 +1,75 @@
+_ = require('underscore');
```

Comment:
perhaps consider `lodash`? it's a drop-in replacement for `underscore` and is often considered to be more performant and robust.